sklearn/base.py

```diff
@@ -292,6 +293,20 @@ def __repr__(self):
         return '%s(%s)' % (class_name, _pprint(self.get_params(deep=False),
                                                offset=len(class_name),),)
 
+    def __getstate__(self):
+        return dict(self.__dict__.items(), __version__=__version__)
```

Comment:
Any idea how to test that? Should I? I guess I could overwrite ``type(self).__module__`` but I'm not sure it's worth it.